extensions/CheckUser/CheckUser.hooks.php

```diff
@@ -275,12 +280,12 @@ public static function maybePruneIPData() {
 	public static function checkUserSchemaUpdates( DatabaseUpdater $updater ) {
 		$base = dirname( __FILE__ );
 
-		$updater->addExtensionUpdate( array( 'CheckUserHooks::checkUserCreateTables' ) );
+		$updater->addExtensionUpdate( [ 'CheckUserHooks::checkUserCreateTables' ] );
 		if ( $updater->getDB()->getType() == 'mysql' ) {
-			$updater->addExtensionUpdate( array( 'addIndex', 'cu_changes',
-				'cuc_ip_hex_time', "$base/archives/patch-cu_changes_indexes.sql", true ) );
-			$updater->addExtensionUpdate( array( 'addIndex', 'cu_changes',
-				'cuc_user_ip_time', "$base/archives/patch-cu_changes_indexes2.sql", true ) );
+			$updater->addExtensionUpdate( [ 'addIndex', 'cu_changes',
+											'cuc_ip_hex_time', "$base/archives/patch-cu_changes_indexes.sql", true ] );
```

Comment:
PhpStorm has failed me :) But it actually doesn't look that bad with 4-spaces per tab